testgrid/config.yaml

```diff
@@ -6830,6 +6832,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gce-scale-correctness
   - name: gce-scale-performance
     test_group_name: ci-kubernetes-e2e-gce-scale-performance
+  - name: gce-private-cluster-correctness-experimental
```

Comment:
I would feel more comfortable if we would introduce a dedicated tab for experimental jobs - I  think we start to have some and more potentially may come, so maybe it's time to do that.

Reply:
Done.